src/SignalR/clients/csharp/Http.Connections.Client/src/Internal/WebSocketsTransport.cs

```diff
@@ -156,6 +156,9 @@ private async Task ProcessSocketAsync(WebSocket socket)
 
                 if (trigger == receiving)
                 {
+                    // Observe exception if there is one to avoid unobserved tasks
+                    _ = receiving.Exception;
```

Comment:
Why don't these exceptions matter? Should we at least be logging them?

Reply:
We probably should, but to no higher than `Debug` since they will include client disconnects.

Reply:
Debug ftw

Reply:
These should never actually have exceptions in them, I was just being paranoid. We have a try catch around both sending and receiving so the only exceptions that could escape would be Pipe.Complete throwing, logger throwing, or `socket.CloseOutputAsync` (this would be the one most likely to throw).

I'll just wrap `socket.CloseOutputAsync` and log it if it throws.

Reply:
Can we get rid of all the `_ = blah.Exception` lines then?

The mere existence of these lines communicates to other developers that these tasks could have completed with an exception. I don't want to risk seeing this pattern copied more widely. The more places this pattern is used, the more likely it will be copied somewhere else.

Reply:
Yeah, I'm good with removing them

Reply:
Cool. If for whatever reason we are still concerned about something unexpected throwing, we could also replace the `_ = blah.Exception` with `await trigger`. That way if there was an unexpected error, WebSocketTransport.StopAsync() would rethrow it.